SP_Doris1115.sql

```diff
@@ -0,0 +1,182 @@
+USE Proj_B1
+
+/*
+1. Write the stored procedure to hire a new employee into an existing position
+ */
+CREATE PROCEDURE newEmployeeExistingPosition
+@FName Varchar(30),
+@LName Varchar(30),
+@DOB DATETIME,
+@Phone Varchar(13),
+@Email Varchar(300),
+@GenderTitle Varchar(50),
+@PSetName Varchar(50),
+@PermAddress Varchar(200),
+@PermCity Varchar(50),
+@PermState char(2),
+@PermZip INT,
+@UWNetID Varchar(100),
+@StudentID INT,
+@PositionName Varchar(50),
+@BeginDate DATE,
+@EndDate DATE
+
+AS
+    DECLARE @EmpID INT, @GenderID INT, @PronounSetID INT, @PosID INT
+    SET @PosID = (SELECT PositionID FROM tblPOSITION WHERE PositionName = @PositionName)
+    SET @GenderID = (SELECT GenderID FROM tblGENDER WHERE GenderTitle = @GenderTitle)
+    SET @PronounSetID = (SELECT PronounSetID FROM tblPRONOUN_SET WHERE PSetName = @PSetName)
+
+    BEGIN TRANSACTION A1
+    INSERT INTO tblEMPLOYEE(Fname, Lname, DOB, Phone, Email, GenderID, PronounSetID, PermAddress,
+                            PermCity, PermState, PermZip, UWNetID, StudentID)
+    VALUES (@Fname, @Lname, @DOB, @Phone, @Email, @GenderID, @PronounSetID, @PermAddress,
+            @PermCity, @PermState, @PermZip, @UWNetID, @StudentID)
+    SET @EmpID = (SELECT SCOPE_IDENTITY())
+
+    INSERT INTO tblEMPLOYEE_POSITION(PositionID, EmployeeID, BeginDate, EndDate)
+    VALUES (@PosID, @EmpID, @BeginDate, @EndDate)
+
+IF @@error <> 0
+    ROLLBACK TRANSACTION
+ELSE
+    COMMIT TRANSACTION
+
```

Comment:
Population of table not required; thanks for writing them out though! You can save these execute statements into a separate file (do this for the second sproc as well) to make the file shorter (and more readable). Other than that, no errors found; good job!